wheatley/main.py

```diff
@@ -85,11 +85,15 @@ def configure_logging():
     logging.getLogger(WaitForUserRhythm.logger_name).setLevel(logging.INFO)
 
 
-def main():
+def main(override_args=None, stop_on_join_tower=False):
```

Comment:
What would it look like of `main()` to always just take the arguments instead of just an override to them?

Reply:
It could work, but `wheatley.main.main()` is the entry point function defined in `setup.py`, so I'd quite like to not have to specify any arguments in there.